utilities/deep_inspection.py

```
@@ -208,25 +236,214 @@ def verify_di_results(conversion: Conversion, vm_name: str) -> None:
+        """Capture DI CR results for one migration status poll.
+
+        Args:
+            status (str): Current migration status from migration polling.
+        """
+        if status != Plan.Status.EXECUTING or DI_RESULTS_KEY in fixture_store:
+            return
+
+        try:
+            conversions = get_plan_conversion_crs(plan_resource=plan, conversion_type="DeepInspection")
+        except ApiException as e:
+            LOGGER.debug(f"Could not query DI CRs during migration: {e}")
```

Ruff (0.15.21)

[warning] Logging statement uses f-string

(G004)